File: Desktop/SummerVacation/0629/python/numEleven.py
euro = input("How much Euro do you have? ")
rate = input("What is the rate? ")
result = float(euro) * float(rate) / 1113.58575
if (float(result) * 1000) - int(float(result) * 100) > 1 :
    print("%.2f Euro is %.2f dollar" %(float(euro),float(result)+0.01))
else :
    print("%.2f Euro is %.2f dollar" %(float(euro),float(result)))

File: Desktop/SummerVacation/0629/src/numEleven.java
import java.util.Scanner;
public class numEleven {
    public static final double RATETO = 1113.58575;
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double euros,rate;
        System.out.print("How many Euros are you exchanging? ");
        euros = scan.nextDouble();
        System.out.print("What is the exchange rate? ");
        rate = scan.nextDouble();
        System.out.printf("%.2f Euros at an exchange rate of %.2f is\n %.2f dollars",euros,rate,euros*rate/RATETO);
    }
}
